frontend/src/app/components/groups/move-group-members/move-group-members.component.html

```diff
@@ -0,0 +1,70 @@
+<div>
+  <h4>{{ headerText }}</h4>
+  <div cdkDropListGroup class="group-member-list">
+    <div
+      *ngFor="let group of groupMembers; let i = index"
+      class="member-list-container"
+    >
+      <div class="member-list-header">
+        <span>{{ group.groupName }}</span>
+        <button
+          mat-icon-button
+          matTooltip="Remove all students from group"
+          (click)="removeAllMembers(i)"
+        >
+          <mat-icon>clear_all</mat-icon>
+        </button>
+      </div>
+      <div
+        cdkDropList
+        [cdkDropListData]="group.members"
+        (cdkDropListDropped)="drop($event)"
+        class="member-list"
+      >
+        <div
```

Comment:
Can we make the squares with student names have a little less vertical padding? There seems to be some extra whitespace in the square above and below their names.

Reply:
The extra whitespace was from the mat-icon-button (the "X" button). I adjusted the padding to compensate.